enzyme/Enzyme/Utils.h

```diff
@@ -1183,6 +1194,10 @@ static inline bool isPointerArithmeticInst(const llvm::Value *V,
       }
     }
 
+  if (isIntelSubscriptIntrinsic(V)) {
+    return true;
+  }
+
```

Comment:
Also add this to getBaseObject ? (which would get the outer memory pointer arg)

Reply:
@wsmoses thanks for the suggestion, I've just added it.